src/install/kubernetes/find-data.mdx

```diff
@@ -0,0 +1,68 @@
+---
+componentType: default
+headingText: Find and use data
+---
+
+
+import kubernetesTestConnection from 'images/kubernetes_screenshot-crop_test-connection.webp'
+
+import kubernetesAddDashboards from 'images/kubernetes_screenshot-crop_add-dashboards.webp'
+
+import kubernetesAddAlerts from 'images/kubernetes_screenshot-crop_add-alerts.webp'
+
+
+Once you've finished the steps, you can test the connection to make sure everything is working properly and view your data.
+
+<img
+    width="80%;"
+    title="Testing Kubernetes installation"
+    alt="Testing Kubernetes installation"
+    src={kubernetesTestConnection}
+/>
+
+
+<Callout variant="tip">
+    Not seeing data? If you completed the steps and are still not seeing data, check out [this troubleshooting page](/docs/kubernetes-pixie/kubernetes-integration/troubleshooting/kubernetes-integration-troubleshooting-not-seeing-data/).
+</Callout>
+
+
+Check out the [Explore your Kubernetes cluster](/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer/) page for instructions on interpreting the data displayed in the Kubernetes navigator, viewing the status of your cluster, identifying entities, accessing the cluster dashboard, and searching for Kubernetes events, and more.
+
+
+Follow these steps to add pre-defined dashboards:
+
+1. Go to <DoNotTranslate>**[one.newrelic.com](https://one.newrelic.com) > +Add data**</DoNotTranslate> and type <DoNotTranslate>**Kubernetes**</DoNotTranslate>.
+
+2. Under <DoNotTranslate>**Dashboards**</DoNotTranslate>, click <DoNotTranslate>**Kubernetes**</DoNotTranslate>.
+
+    <img
+    title="Add Kubernetes dashboards"
+    alt="Add Kubernetes dashboards"
+    src={kubernetesAddDashboards}
+    />
+
+3. Click <DoNotTranslate>**Edit**</DoNotTranslate> if you want to change the account in the popup window that opens.
+
+4. Click <DoNotTranslate>**Setup Kubernetes**</DoNotTranslate> or <DoNotTranslate>**Skip this step**</DoNotTranslate> if you already setup this datasource. 
+
+5. Click <DoNotTranslate>**View dashboard**</DoNotTranslate>, and see your Kubernetes data in New Relic.
+
+
+You can also add pre-defined alert policies:
```

Comment:
Should this context marked as "Optional" and included in a collapsable section?

Reply:
Done.